src/mudlet.pro

```diff
@@ -108,8 +108,16 @@ $$system(git --version, blob, GIT_STATUS)
 isEqual(GIT_STATUS, 0) {
     # Git command appeared to work - so get a description of the current
     # repository state
-    GIT_DESCRIPTION=$$system(git describe --all --dirty --broken)
-    GIT_COMMITHASH=$$system(git rev-parse --short HEAD)
+    GIT_DESCRIPTION = $$(GITHUB_PR_BRANCH)
```

Comment:
This is not the best place to assign this value to this variable, instead it should go into an `else` section - i.e. change line 114 and insert:
```diff
-    }
+    } else {
+        GIT_DESCRIPTION=$$(GITHUB_PR_BRANCH)
+    }
```